notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -6485,6 +6485,39 @@ trust in a potential foe [There was a pointed moment in the film when Claire had
 anti-government sentiment [The domestic terrorist Carroll Oerstadt blew up a passenger ferry in a Timothy McVeigh-like attack on the U.S. Government.]
 
 
+movie: Lifted (2006)
+====================
+
+:: Title
+Lifted
+
+:: Description
+Lifted is a 2006 American computer-animated short science fiction film
+produced by Pixar Animation Studios and directed by Gary Rydstrom.
+
+Synopsis: A young alien attempts to abduct a sleeping farmer.
+
+:: Date
+2006-10-12
+
+:: References
+https://en.wikipedia.org/wiki/Lifted_(2006_film)
+
+:: Ratings
+3 <paul>
+
+:: Major Themes
+alien abduction [The young alien Stu tried and failed to abduct a sleeping farmer.]
+extraterrestrial being [The green, humanoid alien Stu tried to abduct a sleeping farmer under the watchful eye of his gelatinous blog alien examiner, Mr. B.]
+taking a test [The young alien Stu failed his an alien abduction test.]
```

Comment:
"The entirety of the story is about a young alien, named Stu, who fails his alien abduction test (much like young humans fail their driver exams)."

Reply:
? To clarify why it's major, I mean.

Reply:
Good call! That's much less lazy than my original motivation :-) Updated accordingly.